management-api-guide.md

```diff
@@ -631,10 +630,23 @@ Response, if the pre-fill URL has been accessed:
 }
 ```
 
+Below are some _ideas_ for statuses to be returned:
+
+| Status       | Description                                           | Realistic?    | Comment                 |
+| ------------ | ----------------------------------------------------- | ------------- | ----------------------- |
+| `CREATED`    | The PO exists, but nothing has been done with it yet. | ✅ Yes.       |                         |     
+| `OPENED`     | The PO URL has been accessed.                         | ✅ Yes.       |                         | 
+| `SUBMITTED`  | The merchant has clicked "Send" on the PO.            | ✅ Yes.       |                         | 
+| `PENDING`    | Processing has not started yet.                       | Maybe not.    |  We may not be able to provide this.                        | 
+| `PROCESSING` | Processing has started.                               | Maybe not.    |  We may not be able to provide this.                        | 
+| `AWAITING-CUSTOMER` | We are waiting for the merchant to respond.    | Maybe not.    |  We may not be able to provide this.                       |
+| `APPROVED`   | Everything OK: The merchant is approved and active.   | ✅ Yes.       |  Similar to [`GET:/management/v1/merchants/{scheme}/{id}/sales-units`](https://developer.vippsmobilepay.com/docs/APIs/management-api/management-api-guide/#get-the-sales-units-for-a-merchant-by-business-identifier)                       | 
```

Comment:
Isn't this the same as `GET:;/merchants/{scheme}/{id}`, which we already have, and almost just a facade for it?

Reply:
Yes and no... if we talk about it in the context of prefill no, since we have no way of connecting the prefill with the resulting sales unit.

So for a prefill we have the top status-es, for a merchant we have just this (and deactivated), but we have no ids that connect them, so the status-es will will not be valid for the same endpoint, with the same ids.

Reply:
So for this endpoint that gets based on the prefillOrderId, we'll not have this information, because we don't know the resulting sales unit.

Reply:
@niklud OK, so which statuses are we able to provide for pre-fill? POs made without pre-fill are not in scope.

Reply:
```suggestion
| `APPROVED`   | Everything OK: The merchant is approved and active.   | Maybe not.    |  Similar to [`GET:/management/v1/merchants/{scheme}/{id}/sales-units`](https://developer.vippsmobilepay.com/docs/APIs/management-api/management-api-guide/#get-the-sales-units-for-a-merchant-by-business-identifier)                       | 
```